server/app.js

```diff
@@ -14,11 +13,10 @@ app.use(morgan('tiny'));
 app.use(express.static(__dirname + '../client/dist'));
 
 //Routing
-app.use('/reviews', reviews);
-
-const port = process.env.PORT || 3003;
-const server = app.listen(port, () => {
```

Comment:
Oh nice, it looks like you split your server into server + app too? I had to do that to get tests working

Reply:
yes, i tried running the tests as they were, but they didn't pass until i split the server/app, and the db/app.  Also removed routing entirely.